scribble-doc/scribblings/scribble/manual.scrbl

```diff
@@ -1467,16 +1467,20 @@ Examples:
 
 
 @deftogether[(
-@defform[       (defstruct* maybe-link struct-name ([field-name contract-expr-datum] ...)
+@defform[       (defstruct* maybe-link struct-name ([field contract-expr-datum] ...)
                   maybe-mutable maybe-non-opaque maybe-constructor
                   pre-flow ...)]
-@defform/subs[  (defstruct maybe-link struct-name ([field-name contract-expr-datum] ...)
+@defform/subs[  (defstruct maybe-link struct-name ([field contract-expr-datum] ...)
                   maybe-mutable maybe-non-opaque maybe-constructor
                   pre-flow ...)
                ([maybe-link code:blank
                             (code:line #:link-target? link-target?-expr)]
                 [struct-name id
                              (id super-id)]
+                [field field-id
+                       (field-id field-option ...)]
```

Comment:
In the struct implementation, this is valid:

```
(struct foo ([bar]))
```

However, the same in `defstruct*` is not valid:

```
@defstruct*[foo ([(bar) any/c])]
```

Either the implementation of `defstruct[*]` should be adjusted, or the documentation should change `...` with `...+`.

Reply:
Also, should there be checks to make sure that `field-option` is either `#:mutable` or `#:auto`? The current implementation allows any keyword.

Reply:
Thanks for catching this! I fixed `defstruct` to make the empty list of options work.

For the `field-option` checks, it seems like most similar checks in Scribble are syntax errors. I don't know how to do it so I'll leave that out.